checker/manual/external-tools.tex

```diff
@@ -302,17 +302,17 @@ \subsection{Explanation\label{ant-task-explanation}}
 \section{Maven\label{maven}}
```

Comment:
The Checker Framework needs Type Annotations and therefore at least a Java 8 javac - either the standard OpenJDK 8 compiler or the jsr308-langtools compiler.
We can successfully run the Checker Framework on a Java 7 JDK, but need to be able to substitute a Java 8 compiler.
Even if there were an alternative for JavacTask.instance in JDK 7 that would not help us - the JDK 7 javac does not support Type Annotations.
Is the conclusion that in Maven running on JDK 7 we cannot use our own javac?